webpack/assets/javascripts/react_app/components/common/Dropdown.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+const Dropdown = ({ links, id }) => {
+  const listLinks = [];
+
```

Comment:
functional approach to writing this would be:

const listLinks = links.map((link, i) => { 
  let liKey = `notification-link-${i}-${id}`
 return <li key={liKey} id={liKey}>
                    <a target="_blank" href={link.href}>
                    {link.title}</a>
                  </li>
}